#!/usr/bin/env python

PACKAGE = "singaboat_vrx"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("lin_vel_gain", double_t, 0, "Linear velocity gain along X-axis", 1.0, 0, 100)
gen.add("ang_vel_gain", double_t, 0, "Angular velocity gain about Z-axis", 1.0, 0, 100)
gen.add("wamv_half_beam", double_t, 0, "Half beam width of WAM-V", 0.55, 0, 5)

exit(gen.generate(PACKAGE, "singaboat_vrx", "IKSolver"))
